ietf/doc/views_doc.py

```diff
@@ -444,8 +445,21 @@ def document_main(request, name, rev=None):
         else:
             stream_desc = "(None)"
 
-        return render(request, "doc/document_draft.html",
+        html = doc.html_body()
```

Comment:
For this implementation, please guard this with a check on document_html - don't burn the compute if the template isn't going to use the result.